src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -661,6 +661,13 @@ private function registerTranslatorConfiguration(array $config, ContainerBuilder
 
         // Discover translation directories
         $dirs = array();
+
+        foreach ($config['paths'] as $path) {
+            if (is_dir($path)) {
+                $dirs[] = $path;
+            }
```

Comment:
It's not possible to add validation it in the `Configuration` because we need to resolve parameters before checking directories,  so I'll throw an exception here instead.